packages/fixie-hello-world/src/index.tsx

```diff
@@ -1,10 +1,10 @@
-import { ChatCompletion, SystemMessage, UserMessage } from 'ai-jsx/core/completion';
+import { ChatCompletion, SystemMessage, ConversationHistory } from 'ai-jsx/core/completion';
 
-export default function HelloWorld({ message }: { message: string }) {
+export default function HelloWorld() {
   return (
     <ChatCompletion temperature={1}>
-      <SystemMessage>Respond to the user using some variant of the phrase "Hello World!". Be creative!</SystemMessage>
-      <UserMessage>{message}</UserMessage>
+      <SystemMessage>Respond to the user as if you were Clippy from Microsoft Office.</SystemMessage>
```

Comment:
Maybe say "You are Clippy from Microsoft Office. Respond to the user accordingly."